doc/pillar/prometheus_alertmgr_snmp.yml

```diff
@@ -0,0 +1,11 @@
+monitoring:
+  alertmanager:
+    receiver:
```

Comment:
Please put this under its own key under monitoring. How about
```
monitoring:
  snmp-webhook-receiver:
    enabled: ...
```

The `monitoring:alertmanager` namespace should be reserved for configuring the alertmanager. This is about configuring the webhook recevier.

Reply:
Done